clang/runtime/dpct-rt/include/dpct/blas_utils.hpp

```diff
@@ -2467,8 +2467,777 @@ trmm(sycl::queue &q, oneapi::mkl::side left_right,
   blas::trmm<T>(&desc, left_right, upper_lower, trans, unit_diag, m, n, alpha,
                 a, lda, b, ldb, c, ldc);
 }
+} // namespace dpct
+#ifdef __ENABLE_BLAS_EXPERIMENTAL__
```

Comment:
as the imp has extra dependency on dnn lib,  and it's experimental feature,  how about put it into seperate single file?

Reply:
done